src/scripts/main.js

```diff
@@ -1,7 +1,22 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  let message = document.createElement('div');
+  let messageTitle = document.createElement('h2');
+  let messageDescr = document.createElement('p');
+  messageTitle.innerHTML = title;
+  messageDescr.innerHTML = description;
+  message.classList.add('notification', type);
+  messageTitle.className = 'title';
+  document.body.querySelector('h1').after(message);
```

Comment:
The `message` element is appended to the document before its children (`messageTitle` and `messageDescr`) are added. This might cause a brief flicker of an empty notification. Consider appending `message` to the document after adding its children.